torchao/dtypes/uintx/tensor_core_tiled_layout.py

```diff
@@ -33,6 +33,16 @@ def _aqt_is_tensor_core_tile_uint4(aqt):
     )
 
 
+def _aqt_is_tensor_core_tile_int4(aqt):
```

Comment:
need to rename this to remove `tensor_core_tile` in the name I think

Reply:
Changed, and moved out of this file, as indeed it doesn't belong here.